dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -1625,4 +1626,133 @@ def put_student_in_section(student, teacher, script)
   test_user_gets_response_for :show, response: :success, user: :levelbuilder,
     params: -> {script_level_params(@pilot_script_level)},
     name: 'levelbuilder can view pilot script level'
+
+  def create_visible_after_script
+    level = create :maze, name: 'maze 1', level_num: 'custom'
+    script = create :script
+    stage = create :stage, name: 'stage 1', script: script, visible_after: '2020-04-01 08:00:00 -0700'
+    script_level = create :script_level, levels: [level], stage: stage, script: script
+
+    script_level
+  end
+
+  test 'levelbuilder can view level in stage with future visible after date' do
+    Timecop.freeze(Time.new(2020, 3, 27))
+    sign_in @levelbuilder
+
+    script_level = create_visible_after_script
+
+    get :show, params: {
+      script_id: script_level.script,
+      stage_position: script_level.stage.absolute_position,
+      id: script_level.position
+    }
+    assert_response :success
+    Timecop.return
+  end
+
+  test 'levelbuilder can view level in stage with past visible after date' do
+    Timecop.freeze(Time.new(2020, 4, 2))
```

Comment:
this syntax could be problematic when run from different timezones:
```
[staging] dashboard > Time.new(2020, 4, 2)
=> 2020-04-02 00:00:00 +0000
```
```
[development] dashboard > Time.new(2020, 4, 2)
=> 2020-04-02 00:00:00 -0700
```
I would recommend updating this syntax throughout the PR to also specify the timezone.